docker-compose.yml

```diff
@@ -32,9 +34,18 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data/
 
-    expose:
+    ports:
       - ${DB_PORT}
 
+  pgadmin:
```

Comment:
do you use this container? I think that to have one container with postgresql db more than enough